src/modules/bjs_interpreter/interpreter.cpp

```diff
@@ -2,7 +2,10 @@
 #include "core/sd_functions.h"
 #include "core/wifi_common.h"
 #include "core/mykeyboard.h"
-
+#include "core/serialcmds.h"
+#include "modules/rf/rf.h"
+#include "modules/ir/ir_read.h"
+#include "modules/others/bad_usb.h"
```

Comment:
try adding 
`#pragma once`
in the firs line of these header files.. if it works, add them into all .h files